### PR TITLE
Mypy typechecking fix for count.py

### PR DESCRIPTION
**Pull request by ulya-tkch**
- added assert workaround

### CODE REVIEW

File: cleanlab/count.py [hunk 1]
```diff
@@ -113,15 +113,21 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
...
         # Original non-calibrated counts of confidently correctly and incorrectly labeled examples.
-        confident_joint = compute_confident_joint(
-            labels=labels, pred_probs=pred_probs, calibrate=False
+        computed_confident_joint = compute_confident_joint(
+            labels=labels,
+            pred_probs=pred_probs,
+            calibrate=False,
         )
+    else:
+        computed_confident_joint = confident_joint.copy()
```
Comment (jwmueller) [inline cleanlab/count.py:122]: is the `.copy()` truly necessary here?

Comment (ulya-tkch) [reply cleanlab/count.py:122]: Nope! I will remove but I try an avoid all direct assignments without it

**Review by ulya-tkch — COMMENTED** [review]
Removed array copy

File: cleanlab/count.py (hunk 1 above)
Comment (jwmueller) [reply cleanlab/count.py:122]: cool just wanted to make sure my understanding of mypy is not wrong :D

**Review by jwmueller — APPROVED** [review]
LGTM

**Pull request merged**